pkg/component/observability/monitoring/kubestatemetrics/customresourcestate.go

```diff
@@ -182,6 +183,43 @@ func newGardenCustomResourceStateMetrics() customresourcestate.Resource {
 	return resource
 }
 
+func newOperatorExtesionCustomResourceStateMetrics() customresourcestate.Resource {
```

Comment:
nit: Typo (the automated typo check is failing because of this)
```suggestion
func newOperatorExtensionCustomResourceStateMetrics() customresourcestate.Resource {
```